docs/content.rst

```diff
@@ -382,6 +382,40 @@ to allow linking to both generated articles and pages and their static sources.
 
 Support for the old syntax may eventually be removed.
 
+Including other files in your pages
+-----------------------------------
+Both Markdown an reStructuredText syntaxes provide mechanisms for this.
+
+There are some examples for **reStructuredText** using `reStructuredText include`_:
+
+    .. code-block:: none
+
+        .. include:: file.rst
+
+Including a fragment of a file delimited by two identifiers, highlighted as C++ (slicing based on line numbers is also possible):
+
+    .. code-block:: none
+
+        .. include:: main.cpp
+            :code: c++
+            :start-after: // begin
+            :end-before: // end
+
+Including a raw HTML file (or an inline SVG or whatnot) and put it directly into the output without any processing:
```

Comment:
I didn't expect that my hastily-written comment becomes *the* official documentation :sweat_smile: -- the `whatnot` is maybe a bit too much.

Reply:
Sure, I'll remove that ;)